activerecord/lib/active_record/model_schema.rb

```diff
@@ -388,6 +388,13 @@ def column_names
         @column_names ||= columns.map(&:name)
       end
 
+      def symbol_column_to_string(name_symbol) # :nodoc:
+        @symbol_column_to_string_name_hash ||= column_names.each_with_object({}) do |name, hash|
```

Comment:
```ruby
@symbol_column_to_string_name_hash ||= column_names.index_by(&:to_sym)
```

Reply:
It's cleaner. I benchmarked it and it's just as fast. I'll update to this method 

```
column_names = ["id", "name", "user_name", "issues_count", "language", "description", "full_name", "notes", "created_at", "updated_at", "github_error_msg", "commit_sha", "stars_count", "subscribers_count"].freeze

require 'benchmark/ips'

Benchmark.ips do |x|
  x.report("each_with_object") {
    column_names.each_with_object({}) do |name, hash|
      hash[name.to_sym] = name
    end
  }
  x.report("index_by") {
    column_names.index_by(&:to_sym)
  }
  x.compare!
end
# Warming up --------------------------------------
#     each_with_object    23.947k i/100ms
#             index_by    23.890k i/100ms
# Calculating -------------------------------------
#     each_with_object    256.539k (± 3.4%) i/s -      1.293M in   5.046333s
#             index_by    257.835k (± 5.3%) i/s -      1.290M in   5.018684s

# Comparison:
#             index_by:   257835.3 i/s
#     each_with_object:   256539.1 i/s - same-ish: difference falls within error
```